### PR TITLE
refactor(http): import core dependenciess from angular2/core

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**

### CODE REVIEW

**Comment by pkozlowski-opensource** [comment]
@vsavkin could you please have a look? I guess http shouldn't need anything else than core.

File: modules/angular2/src/http/backends/xhr_backend.ts [hunk 1]
```diff
@@ -3,10 +3,10 @@ import {ReadyStates, RequestMethods, ResponseTypes} from '../enums';
 import {Request} from '../static_request';
 import {Response} from '../static_response';
 import {ResponseOptions, BaseResponseOptions} from '../base_response_options';
-import {Injectable} from 'angular2/angular2';
+import {Injectable} from 'angular2/core';
 import {BrowserXhr} from './browser_xhr';
 import {isPresent} from 'angular2/src/facade/lang';
-import {Observable} from 'angular2/angular2';
+import {Observable} from 'angular2/core';
```
Comment (robwormald) [inline modules/angular2/src/http/backends/xhr_backend.ts]: maybe combine Injectable and Observable into a single import here

File: modules/angular2/src/http/http.ts [hunk 2]
```diff
@@ -1,12 +1,12 @@
 import {isString, isPresent, isBlank} from 'angular2/src/facade/lang';
 import {makeTypeError} from 'angular2/src/facade/exceptions';
-import {Injectable} from 'angular2/angular2';
+import {Injectable} from 'angular2/core';
```
Comment (robwormald) [inline modules/angular2/src/http/http.ts]: also here

**Comment by robwormald** [comment]
FYI this should also close https://github.com/angular/angular/issues/5333

**Comment by pkozlowski-opensource** [comment]
This is already fixed in master, closing.

**Pull request closed without merging**